packages/block-editor/src/components/block-popover/index.js

```diff
@@ -88,6 +93,20 @@ function BlockPopover(
 
 		return {
 			getBoundingClientRect() {
+				if ( isZoomOut() ) {
+					const selectedRect =
+						getVisibleElementBounds( selectedElement );
+					const parentRect = getVisibleElementBounds(
+						selectedElement.parentElement
+					);
```

Comment:
Is `parentRect` the element represented by `sectionRootClientId`? I'm not immediately clear from the code what the "parent" relationship is.

Can you clarify for me?

Reply:
Yes it's the container which it is the section root. It's the only way I found to get a rect with the correct left and right.

Reply:
In which case I wonder if the code could be made more self documenting?

```suggestion
					const sectionRootElementRect = getVisibleElementBounds(
						selectedElement.parentElement
					);
```